pkg/printers/internalversion/printers_test.go

```diff
@@ -5757,35 +5757,57 @@ func TestPrintLease(t *testing.T) {
 }
 
 func TestPrintPriorityClass(t *testing.T) {
+	preemptNever := api.PreemptNever
```

Comment:
PreemptionPolicy is an optional field. Could you please add a test case without PreemptionPolicy.

Reply:
thanks ! done

Reply:
Where is the test case `PreemptionPolicy: nil`?

Reply:
Sorry for the mistake. added